src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
+/* eslint-disable no-useless-constructor */
 export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number | never;
+}
 
+function numberRound(x: number): number {
+  return Math.floor(x * 100) / 100;
 }
 
-export class Triangle {
+export enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
 
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  public shape = Shape.triangle;
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0
+      || Math.max(a, b, c) >= (a + b + c - Math.max(a, b, c))) {
```

Comment:
it isn't a mistake, but you can improve a bit your code. You should check if all arguments are positive in all cases. So you can create a function wich will check it and return the boolean value